tools/topology/CMakeLists.txt

```diff
@@ -92,9 +92,11 @@ set(TPLGS
 	"sof-apl-asrc-wm8804\;sof-apl-asrc-wm8804"
 	"sof-apl-asrc-pcm512x\;sof-apl-asrc-pcm512x"
 	"sof-apl-src-pcm512x\;sof-apl-src-pcm512x"
-	"sof-cml-rt5682\;sof-cml-rt5682\;-DPLATFORM=cml"
-	"sof-cml-rt5682\;sof-whl-rt5682\;-DPLATFORM=whl"
-	"sof-cml-rt5682\;sof-icl-rt5682\;-DPLATFORM=icl"
+	"sof-cml-rt5682\;sof-cml-rt5682\;-DPLATFORM=cml\;-DCPROC=volume"
+	"sof-cml-rt5682\;sof-cml-eq-rt5682\;-DPLATFORM=cml\;-DCPROC=eq-fir-volume"
+	"sof-cml-rt5682\;sof-cml-eq-lowpass-rt5682\;-DPLATFORM=cml\;-DCPROC=eq-fir-volume\;-DFILTERSET=lowpass"
```

Comment:
Good, the XPROC for passing the pipeline macro is useful. I see the name CPROC infers capture processing but we should be able to separate DMIC, speaker, headset mic, headset ear endpoints and possibly define processing for several of them simultaneously. Could they be DMICPROC, SPKPROC, HSMICPROC, HSEARPROC?

In this case value lowpass for FILTERSET should be e.g. eq_fir_coef_lowpass_7khz_40db_48khz to call out the applied processing and simplify the generic pipeline macro. I wonder if the name usage could be like HSMICFILT=eq_fir_coef_xxxx. It would add m4 code to topology top level to redirect the processing to correct pipeline. We don't know pipeline numbers at makefile level so this kind of abstraction seems necessary.